public/tpl/footer.html

```diff
@@ -8,6 +8,10 @@ <h2>About</h2>
                 <span ng-bind-html="settings.poweredbyCustom" ng-if="settings.poweredbyCustom">{{settings.poweredbyCustom}}</span>
                 <br>
                 <a href="https://www.mintme.com/">monetize traffic - mintMe</a>
+                <span>
+                    <i class="fas fa-cookie-bite"></i>
+                    {{settings.notificationCookie}}
```

Comment:
Just place a text here. It's not a sensitive content to place it in separate variable

Reply:
Done

Reply:
> Just place a text here. It's not a sensitive content to place it in separate variable

Done